internal/backend/db/dbgorm/dbgorm.go

```diff
@@ -76,6 +76,14 @@ func (db *gormdb) Connect(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("opendb: %w", err)
 	}
+	sqlDB, _ := client.DB()
+
+	if db.cfg.MaxOpenConns > 0 {
+		sqlDB.SetMaxOpenConns(10)
```

Comment:
Why do you ignore the provided value? 

Set 10 as default and use the provided value if provided

Make sure it is non negative

Reply:
oops.. left from testing. fixed